src/components/hooks/useSelect.tsx

```diff
@@ -91,16 +94,19 @@ function getItemRenderer<T>(value: T, options: SelectOptions<T>) {
     { handleClick, handleFocus, modifiers, index },
   ) => {
     const label = getLabel(item, options);
+    const { renderItem } = options;
+    const isSelected = selectedLabel === label;
+    const { active, disabled } = modifiers;
     return (
       <MenuItem
-        active={modifiers.active}
-        disabled={modifiers.disabled}
-        selected={selectedLabel === label}
+        active={active && isSelected}
```

Comment:
This breaks the component in a bad way.

We no longer know which item is active when using the keyboard to navigate between items.